syft/source/stereoscope/provider.go

```diff
@@ -0,0 +1,76 @@
+package stereoscope
+
+import (
+	"context"
+
+	"github.com/anchore/stereoscope"
+	"github.com/anchore/stereoscope/pkg/image"
+	"github.com/anchore/stereoscope/tagged"
+	"github.com/anchore/syft/syft/source"
+)
+
+const ImageTag = "image"
+
+type stereoscopeSourceProvider struct {
```

Comment:
```suggestion
type Provider struct {
```
given the suggested package name, then it reads as `stereoscopesource.Provider`